asreviewcontrib/dataset_name/your_dataset.py

```diff
@@ -1,29 +1,60 @@
 from asreview.datasets import BaseDataSet
 from asreview.datasets import BaseDataGroup
+from pathlib import Path
 
-class YourDataGroup(BaseDataGroup):
-    group_id = "your_data_group"
-    description = "A new data group with my awesome datasets."
+
+class example_dataset_local(BaseDataSet):
+    """
+    This is an example dataset that is stored locally.
+    """
+
+    import os
+    __filepath__ = os.getcwd()
 
     def __init__(self):
+        super().__init__(
+            dataset_id="example_dataset_local",
+            filepath=str(Path(self.__filepath__, 'data', 'your_dataset.csv')),
```

Comment:
Can't you use `Path(__file__).parent`?

Reply:
We couldn't since the setup only installed namespace packages and thus didn't import datafiles. I've adjusted this now.